app/scenes/Collection.js

```diff
@@ -207,8 +204,62 @@ class CollectionScene extends React.Component<Props> {
                   </React.Fragment>
                 )}
 
-                <Subheading>Recently edited</Subheading>
-                <DocumentList documents={recentDocuments} limit={10} />
+                <Tabs>
```

Comment:
Using the same pattern here as the `/dashboard` routes that have worked pretty well